### PR TITLE
use sceClib memory functions

### PR DESCRIPTION
**Pull request by frangarcj**

### CODE REVIEW

File: newlib/libc/machine/arm/memcpy-stub.c [hunk 1]
```diff
@@ -29,7 +29,13 @@
 
 #include "acle-compat.h"
 
-#if (defined (__OPTIMIZE_SIZE__) || defined (PREFER_SIZE_OVER_SPEED))
+#if defined (__vita__)
+# include <psp2/kernel/clib.h>
+void memcpy (void *dest, const void *source, size_t n){
```
Comment (isage) [inline newlib/libc/machine/arm/memcpy-stub.c]: `void *`

File: newlib/libc/machine/arm/memmove.c [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#if defined (__vita__)
+# include <psp2/kernel/clib.h>
+void memmove (void *dest, const void *source, size_t n){
```
Comment (isage) [inline newlib/libc/machine/arm/memmove.c]: `void *`

File: newlib/libc/machine/arm/memset.c [hunk 3]
```diff
@@ -0,0 +1,8 @@
+#if defined (__vita__)
+# include <psp2/kernel/clib.h>
+void memset (void *dest, int c, size_t n){
```
Comment (isage) [inline newlib/libc/machine/arm/memset.c]: `void *`

**Review by d3m3vilurr — CHANGES_REQUESTED** [review]
indentations have to be hardtab :)

**Pull request merged**